cmake/parthenonConfig.cmake.in

```diff
@@ -0,0 +1,26 @@
+#=========================================================================================
+# (C) (or copyright) 2020. Triad National Security, LLC. All rights reserved.
+#
+# This program was produced under U.S. Government contract 89233218CNA000001 for Los
+# Alamos National Laboratory (LANL), which is operated by Triad National Security, LLC
+# for the U.S. Department of Energy/National Nuclear Security Administration. All rights
+# in the program are reserved by Triad National Security, LLC, and the U.S. Department
+# of Energy/National Nuclear Security Administration. The Government is granted for
+# itself and others acting on its behalf a nonexclusive, paid-up, irrevocable worldwide
+# license in this material to reproduce, prepare derivative works, distribute copies to
+# the public, perform publicly and display publicly, and to permit others to do so.
+#=========================================================================================
+
+# Favor using the kokkos package that was built with parthenon, if one has not been specified 
+if ( NOT Kokkos_FOUND )
+  find_package(Kokkos 3 REQUIRED PATHS @CMAKE_INSTALL_PREFIX@)
+  set(Kokkos_BUILT_WITH_CUDA @Kokkos_ENABLE_CUDA@)
```

Comment:
Use `kokkos_check(DEVICE CUDA)` here, as the detected version might be different from the version partenon was built with.

Reply:
`kokkos_check` will throw a fatal error if the found version of Kokkos isn't built with cuda, but this is a use case we still want to support, right? (For example, if I want to do software development on a machine with only OpenMP)

Reply:
Yes, we want the ability to use Kokkos without CUDA.